src/components/EpicView/helpers/epicViewHelpers.ts

```diff
@@ -0,0 +1,12 @@
+import { Issue } from "../../../../types";
+
+export function filtersearch(query :string, epics : Issue[]) {
+  const searchstring = query.toLowerCase().trim();
+  return epics.filter((item) => item.summary.toLowerCase().includes(searchstring)
+    || searchstring === ""
```

Comment:
```suggestion
    || searchstring === ""
```
This should be the first condition or even an early return.